be1-go/message/messagedata/chirp_add.go

```diff
@@ -6,13 +6,16 @@ import (
 
 // ChirpAdd defines a message data
 type ChirpAdd struct {
-	Object    string `json:"object"`
-	Action    string `json:"action"`
-	Text      string `json:"text"`
-	Timestamp int64  `json:"timestamp"`
+	Object string `json:"object"`
+	Action string `json:"action"`
+	Text   string `json:"text"`
+
+	//Timestamp is a Unix timestamp
+	Timestamp int64 `json:"timestamp"`
 }
 
 // Verify verifies that the ChirpAdd message is correct
+// Implements VerifiableMessageData
```

Comment:
Should start with:

```
Verify implements VerifiableMessageData. It ...
```

This is a convention in our code base that we use. See https://dedis.github.io/dela/#/guideline?id=comments.

Reply:
Sorry, I thought that since the verify was on the first line it was redundant. I'll change it